### ISSUE TITLE
NVDA announces the word "table" quite repeatedly upon reaching headings, landmarks and frames

### ISSUE THREAD

**Issue by amirsol81** [nvaccess/nvda#11489]
With Google Chrome and on some pages and websites, NVDA says, "table table table table", or, "out of table out of table out of table out of table", upon moving around text, reaching headings, many links, etc.
### Steps to reproduce:
1. Having made sure the "Data table" check box is unchecked in the Browse tab of NVDA's Settings dialog, open the following page : https://digital.nepr.net/podcasthub/word-matters
2. Try navigating around via headings or try pressing Control+Home first and then navigating around via headings, landmark, frame etc.
### Actual behavior:
We'll hear the word "table" a lot and annoyingly upon reaching tables, frames, landmarks, etc. The same is true about the phrase "out of table". It's also duplicable with the "Data tables" check box checked, so it seems that it has no effect although it seems to me that the tables here, or most of them, should be detected as data tables so as to suppress the annoying announcements. This is with Google Chrome 84, and NVDA doesn't exhibit this behavior with Internet Explorer doesn't Explorer.
### Expected behavior:
NVDA shouldn't say "table" or "out of table" repeatedly.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
Both Alpha 20725 and 2020.2.
#### Windows version:
Win 10 Pro 64-Bit, Version 2004
#### Name and version of other software in use when reproducing the issue:
Google Chrome 84.0.4147.125. It's not duplicable with Internet Explorer 11. I don't have access to Firefox for testing. For whatever it's worth, JAWS doesn't exhibit this issue with Chrome either.
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Mentioned above -- both the latest available alpha and the latest available official release
#### If addons are disabled, is your problem still occuring?
Yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Yes -- to no avail

**Comment by Adriani90** [comment]
cc: @feerrenrut

**Comment by amirsol81** [comment]
This has been resolved so I'm closing it.
